test/unittest/unittest_databuffer_file.cpp

```diff
@@ -0,0 +1,208 @@
+/**
+ * Copyright (C) 2020 Samsung Electronics Co., Ltd. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @file        unittest_nntrainer_databuffer_file.cpp
+ * @date        10 April 2020
+ * @brief       Unit test databuffer from file.
+ * @see         https://github.com/nnstreamer/nntrainer
+ * @author      Jihoon Lee <jhoon.it.lee@samsung.com>
+ * @bug         No known bugs
```

Comment:
may be leave the original author's name.

Reply:
Got it and fixed! Thanks for letting me know :)

Reply:
Add your name & email addr.